orion/sites/admin/src/achieve/achieveEdit.styl

```diff
@@ -0,0 +1,49 @@
+@import "../static.styl"
+
+#view-achieve-edit
+    width: spacing96x
+    padding: spacing1x spacing8x spacing4x
+    box-sizing: border-box
+
+    & h2
+        margin: spacing4x 0
+    & h4
+        margin: 0
+
+    & input
+        width: 100%
+        height: spacing3x
+        margin: spacing1x 0 spacing2x
+        font-size: fontSize0
+
+    & textarea
+        margin: spacing1x 0 spacing2x
+        width: 100%
+        height: spacing12x
+        border: 1px solid colorGrayLight
+        line-height: lineHeight0
+        font-size: fontSize0
+
+    & .buttons
+        margin-top: spacing2x
+        margin-bottom: spacing8x
+
+    & button
+        padding: spacing2x spacing4x
```

Comment:
I think order for CSS should be padding, border, border-radius, outline, font-size, cursor